tools/quantwiz_iq/quantwiz_iq.xml

```diff
@@ -0,0 +1,170 @@
+        
+        <conditional name="input_purity_file">
+            <param name="input_purity_file_option" type="select" display="radio" label="Use default or from the history">
+                <option value="default" selected="true">Use Default</option>
+                <option value="history">Use file from the history</option>
+            </param>
+            <when value="default">
+                <param name="purity_type" type="select" label="Select default purity correction type">
+                    <option value="iTRAQ4plex.tsv">iTRAQ 4plex</option>
+                    <option value="iTRAQ8plex.tsv" selected="true">iTRAQ 8plex</option>
+                    <option value="TMT6plex.tsv">TMT 6plex</option>
+                    <option value="TMT10plex.tsv">TMT 10plex</option>
+            </param>
+            </when> 
+            <when value="history">
+                <param type="data" name="purity_type" format="txt" label="Select purity correction file from the history"/>
+            </when> 
+        </conditional>
+        
+    </inputs>
+    
+    <outputs>
+        <data name="quant" format="tabular" label="${tool.name}_Quant.tabular" from_work_dir="${input_type.input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}.tsv"/>
+        <data name="noquant" format="tabular" label="${tool.name}_No_Quant.tabular" from_work_dir="${input_type.input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}_noquant.tsv"/>
+        <data format="txt" name="iq_params" label="${tool.name}_QuantWiz_IQ_parameters"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="file_ext" value="mgf"/>
+            <param name="input_file" value="Test_Orbi_iTRAQ_8plex.mgf" ftype="mgf"/>
+            <param name="label_type" value="iTRAQ"/>
+            <param name="num_plex" value="8plex"/>
+            <param name="min_reporters" value="1"/>
+            <param name="area_method" value="S"/>
+            <param name="base_reporter" value="113"/>
+            <param name="normalization" value="no"/>
+            <param name="peak_intensity_threshold" value="0.0"/>
+            <param name="tol_unit" value="Da"/>
+            <param name="min_tol" value="0.05"/>
+            <param name="max_tol" value="0.05"/>
+            <param name="input_purity_file_option" value="default"/>
+            <param name="purity_type" value="iTRAQ8plex.tsv"/>
+            <output name="quant" file="Test_Orbi_iTRAQ_8plex_S_0.05.tsv">
+                <assert_contents>
+                        <has_text text="Retention_time" />
+                </assert_contents>
+            </output>
+            <output name="noquant" file="Test_Orbi_iTRAQ_8plex_S_0.05_noquant.tsv">
+                <assert_contents>
+                    <has_text text="Base_Intensity" />
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+        **QuantWiz_IQ**
+         in Galaxy for Isobaric Quantitation
```

Comment:
can you please extend this help substantially?
Galaxy can be removed, but what type of input, output, what does this tool etc is important ...